gitlab/v4/objects/releases.py

```diff
@@ -25,6 +25,10 @@ class ProjectReleaseManager(CRUDMixin, RESTManager):
     _create_attrs = RequiredOptional(
         required=("tag_name", "description"), optional=("name", "ref", "assets")
     )
+    _list_filters = (
+        "order_by",
+        "sort",
+    )
```

Comment:
Thanks a lot @nickbroon, I'm surprised `black` didn't complain here to go with a single line but this will have an easier diff in the future anyway ;)